system/vrops-custom-importer/templates/job.yaml

```
@@ -0,0 +1,45 @@
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: cron-vrops-custom-importer
+  namespace: {{ .Values.namespace }}
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade,pre-rollback
+    "helm.sh/hook-delete-policy": before-hook-creation
+    "helm.sh/hook-weight": "3"
+spec:
+  schedule: "{{ .Values.schedule }}"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels:
+            app: vrops-custom-importer
+        spec:
+          containers:
+          - name: {{ .Values.image.name}}
+            image: {{ .Values.image.registry}}
+            env:
+              - name: REGION
+                value: {{ .Values.region }}
```

```suggestion
                value: {{ .Values.global.region }}
```

This is a default and can be mapped. Make sure you add it to values.yaml too, just to make it clear that this is referred to know which files to map in the pipeline.